app/components/README.md

```diff
@@ -0,0 +1,19 @@
+## Rails view components
+JS and SCSS files are optional
+```
+/app
+  /components
+    /my_component
+      my_component.js
+      my_component.test.js
+      my_component.scss
+      my_component.html.haml
+    my_component.rb
```

Comment:
Do we want the .rb in the directory, too? (Saving vertical visual space)

Reply:
my thinking was thats in an entry point to using the component

Reply:
in node.js you can have an index file that mean you only need to reference the folder. if can do this consistently with rails too i would favour what you suggest here @david-mears-dfe

Reply:
"thats in an entry point to using the component"

I'm not sure what you mean?

I'm not too picky about this.